generator/src/test/java/com/scottlogic/deg/generator/generation/RegexStringGeneratorTests.java

```diff
@@ -279,20 +276,22 @@ private void expectOrderedResults(String... expectedValues) {
         StringGenerator generator = constructGenerator(true);
 
         List<String> generatedValues = new ArrayList<>();
-        for (String generatedValue : generator.generateAllValues()) {
-//            System.out.println("generated: " + generatedValue);
-            generatedValues.add(generatedValue);
+        generator.generateAllValues().iterator().forEachRemaining(generatedValues::add);
+
+        Assert.assertEquals(expectedValues.length, generatedValues.size());
+        for (int i = 0; i < expectedValues.length; i++) {
+            Assert.assertEquals(expectedValues[i], generatedValues.get(i));
         }
+    }
 
-        String actual = generatedValues.get(0);
-        String expected = Arrays.asList(expectedValues).get(0);
+    private void expectAnyOrder(String... expectedValues) {
+        StringGenerator generator = constructGenerator(true);
 
-//        System.out.println("actual: " + actual);
-//        System.out.println("expected: " + expected);
+        List<String> generatedValues = new ArrayList<>();
+        generator.generateAllValues().iterator().forEachRemaining(generatedValues::add);
```

Comment:
Same as above, can we use collectors

```suggestion
        List<String> generatedValues = generator.generateAllValues().stream().collect(Collectors.toList());
```